testing/php7-memcached/APKBUILD

```diff
@@ -5,7 +5,7 @@ _pkgreal=memcached
 _pkgrepo=php-memcached
 _pkgbranch=php7
 _pkgcommit=583ecd68faec886ac9233277531f78fb6e2043c7
-pkgver=3.0.0_pre160808
+pkgver=3.0.0_pre1
```

Comment:
@jirutka Just found out that the checksums are missing and they were hardcoded to the previous version number format. Fixing now.